firmware/README.md

```diff
@@ -0,0 +1,102 @@
+# Juicy Switch Firmware 🔌 🍊 💻
+
+⚠️ **This is work in progress!** ⚠️
+
+The firmware powering Juicy's *esp* chip. It lets you configure the WiFi that Juicy should connect to and provides a RESTful API for controlling the chip's LED. Later, when Juicy is fully assembled, you will be able to power the socket on and off via that API.
```

Comment:
Should we call it socket or switch? What do you prefer?

Reply:
Let's do *switch*. Sounds easier.